GitOut/Features/Git/Log/GitLogPage.xaml

```diff
@@ -202,6 +202,77 @@
                             <Path Fill="{DynamicResource MaterialForegroundBase}" Data="{StaticResource ContentCopy}"/>
                         </Button>
                         <Border Style="{StaticResource MaterialVerticalDivider}"/>
+                        <ToggleButton x:Name="Remotes" Style="{StaticResource MaterialIconToggleButtonStyle}" ToolTip="Remotes">
+                            <Path Fill="{DynamicResource MaterialForegroundBase}" Data="{StaticResource DownloadNetworkOutline}"/>
+                        </ToggleButton>
+                        <Popup
+                            StaysOpen="False"
+                            AllowsTransparency="True"
+                            IsOpen="{Binding IsChecked, ElementName=Remotes}"
+                            PopupAnimation="Fade"
+                            Placement="Bottom"
+                        >
+                            <Grid>
```

Comment:
outer grid is unused and can probably be removed

Reply:
Sadly no. The Border's padding property is not respected by Popup (I think) so the content is offset incorrect without the grid :-/